### ISSUE TITLE
Error when trying to log in

### ISSUE THREAD

**Issue by cr0un** [Android1500/AndroidFaker#67]
Error when trying to log in:
Certificate pinning failure!Peer certificate chain:sha256/eHrz3pbcSd98uEFiAjVa9mcajRs52UzLB3LYUnIT9gQ=:CN=android1500.appsha256/NYbU7PBwV4y9J67c4guWTki8FJ+uudrXL0a4V4aRcrg=: CN=E5,0=Let'sEncrypt,C=USsha256/C5+|pZ7tcVwmwQ|McRtPbsQtWLABXhQzejna0wHFr8M=: CN=ISRG RootX1,0=Internet Security Research Group, C=USPinned certificates for android1500.app:sha256/Ee|780QLEjleVSEWQN3o0+fWfmDUNPTUv8Z76B9rPeg=

<img width="616" alt="image" src="https://github.com/user-attachments/assets/cddca3a9-cc33-4e8c-b613-4d027463a9ae">

**Comment by cr0un** [comment]
Application version v2.0.0-beta-2
Tried it on different devices, reinstalling didn't help, rebooting didn't help either.
Android 14

**Comment by cr0un** [comment]
On v1.9.2 there is no such error, apparently something with the authorization server for the beta version.

**Comment by Android1500** [comment]
Strange issue it should be not give this error lemme check to my server

**Comment by Kosko33** [comment]
Yeah the same for me cant login, but on the old one works normally

**Comment by contidz** [comment]
![Screenshot_2024-11-30-11-51-34-49_b32194d3714b2caa39157afea762e54b](https://github.com/user-attachments/assets/d964f0c4-3df4-4b9d-8613-9eeab1b2bd13)
login error

**Comment by cr0un** [comment]
@Android1500 Hi, any news?

**Comment by contidz** [comment]
@Android1500 fix sever beta ?

**Comment by Android1500** [comment]
> @Android1500 fix sever beta ?

I was on a trip but now i m back from trip and at my home so i will going to fix it tmrw

**Comment by mrnobody1907** [comment]
Same, good luck

**Comment by mrnobody1907** [comment]
Need for work vip functions:( soon?

**Comment by ljorge705** [comment]
Same issue :/

**Comment by contidz** [comment]
@Android1500 Have you fixed the beta error yet? I've been waiting for a long time

**Comment by Android1500** [comment]
working on it now

> @Android1500 Have you fixed the beta error yet? I've been waiting for a long time

**Comment by mrnobody1907** [comment]
:(

**Comment by marschix3** [comment]
Please fix this s**t ASAP. I paid for it and have not been able to log in for 3 weeks!

**Comment by Android1500** [comment]
Going to release on Christmas next beta

**Comment by snakechilds** [comment]
waiting

**Comment by pippo8883** [comment]
still waiting

**Comment by snakechilds** [comment]
news ?

**Comment by cr0un** [comment]
hi, any news?)

**Comment by Gulliver688** [comment]
> Going to release soon next beta

Hi! First you wrote that next beta will be after Christmas, and then you edited your post) 

So, are there any news?

**Comment by mrnobody1907** [comment]
??????

**Comment by Android1500** [comment]
> > Going to release soon next beta
> 
> 
> 
> Hi! First you wrote that next beta will be after Christmas, and then you edited your post) 
> 
> 
> 
> So, are there any news?

I have some medical issues thats why i postpone release of next beta its need some minor code changes before release and now i m feel better so i am working on it.

**Comment by septiari02** [comment]
> > > Going to release soon next beta
> > 
> > 
> > Hi! First you wrote that next beta will be after Christmas, and then you edited your post)
> > So, are there any news?
> 
> I have some medical issues thats why i postpone release of next beta its need some minor code changes before release and now i m feel better so i am working on it.

I have the same problem as this thread, I bought vip on the version 2 beta 5 days ago and until now the error alert is still the same when registering or logging in. I hope you can fix this problem quickly, and may I know when you will release the next version?

**Comment by howlongcanimakemyusername** [comment]
It's beta you guys. Give 1500 a break. They do good work, and are just one person. Go back to 192 if you don't like beta testing.

**Comment by cr0un** [comment]
> It's beta you guys. Give 1500 a break. They do good work, and are just one person. Go back to 192 if you don't like beta testing.

Personally I have beta works more stable, 192 often crashes with error (this module is not enabled) and this is not on 1 device, but on several, so I'm waiting for a solution to this problem. (it is quite strange, because the error on the authorization server) I also have several accounts with VIP status for 192 and for the beta version, so for me personally it is important.

**Comment by bmnltt** [comment]
Hello everyone have we experienced a solution to this problem?

**Comment by cr0un** [comment]
@bmnltt no, we're awaiting

**Comment by bmnltt** [comment]
> [@bmnltt](https://github.com/bmnltt) no, we're awaiting

Very well, as am I. Thanks.

**Comment by milkyshake38** [comment]
hallo? any update? still the same problem :(

**Comment by Android1500** [comment]
Fixed in latest beta
